README.md

```diff
@@ -42,26 +42,21 @@ Download a copy of the latest Regrader: https://github.com/fushar/regrader/relea
 
 #### 3. Configure Regrader
 
-Create an InnoDB database in your MySQL database server. Then, open the database configuration file **application/config/database.php** and add your hostname, username, password, and database name in this part of the file:
-
-```php
-$db['default'] = array(
-    //
-    'hostname' 		=> 'localhost',
-    'username' 		=> '',
-    'password' 		=> '',
-    'database' 		=> '',
-    //
-);
-```
+Create an InnoDB database in your MySQL database server.
 
-Next, open the configuration file **application/config/config.php** and add a random string in this part of the file:
+Then, make a copy of `.env` from `.env.example`:
 
-```php
-$config['encryption_key'] = '';
+```bash
+$ cp .env.example .env
 ```
 
-It is advisable that the string consists of 32 random characters.
+And edit each entry of `.env` based on your system configuration:
+
+- `ENCRYPTION_KEY` denotes the encryption key that will be used by CodeIgniter. It is advisable that the string consists of 32 random characters.
+- `DB_HOSTNAME` denotes the hostname of the MySQL database.
+- `DB_USERNAME` and `DB_PASSWORD` denote the credentials of the MySQL user related to the database.
+- `DB_NAME` denotes the database name that will be used by Regrader.
+- `CPP_EXECUTABLE_PATH` denotes the C++ compiler path that will be used to compile checker source code (e.g., `usr/bin/g++`).
```

Comment:
This is only for compiling checker source code.
The compiler path used in grading would be editable via Web UI.